draft-ietf-quic-recovery.md

```diff
@@ -877,12 +877,14 @@ similar to a sender's response on a Retransmission Timeout (RTO) in TCP
 
 ## Pacing {#pacing}
 
-This document does not specify a pacer, but it is RECOMMENDED that a sender pace
-sending of all in-flight packets based on input from the congestion
-controller.  Sending multiple packets into the network without any delay between
-them creates a packet burst that might cause short-term congestion and losses.
-Implementations MUST either use pacing or another method to limit such bursts
-to the initial congestion window; see {{initial-cwnd}}.
+A sender SHOULD pace sending of all in-flight packets based on input from the
+congestion controller.
+
+Sending multiple packets into the network without any delay between them creates
+a packet burst that might cause short-term congestion and losses. Senders MUST
+either use pacing or limit such bursts. Senders SHOULD limit bursts to the
+initial congestion window; see {{initial-cwnd}}. A sender with knowledge that
```

Comment:
I don't think we should refer to deep buffers. At high data rates in DC the path can process packets at multi-Gbps line rate so even with reasonable buffer sizes the bursts can be absorbed.  How about:

"A sender with knowledge that the network path to the receiver can absorb larger bursts may use a higher limit."

Reply:
Done.